packages/core/src/divider/Divider.css

```diff
@@ -24,4 +24,5 @@
 .saltDivider-vertical {
   width: var(--salt-size-border);
   height: auto;
+  align-self: center;
```

Comment:
This change breaks vertical dividers in a flex layout

Reply:
removed